.github/workflows/test.yml

```diff
@@ -3,7 +3,15 @@
 
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - dev
+
+  pull_request:
+    branches:
+      - dev
```

Comment:
add `main` here as well.
I know that with the current release model we don't PR to `main` but there is no reason to remove it.